womtool/src/main/scala/womtool/cmdline/WomtoolCommandLineParser.scala

```diff
@@ -80,10 +80,9 @@ class WomtoolCommandLineParser extends scopt.OptionParser[PartialWomtoolCommandL
     .action((_, c) => c.copy(command = Option(Graph)))
     .text("Generate and output a graph visualization of the workflow in .dot format" + System.lineSeparator)
 
-  // Undocumented until fully-working
-  //  cmd("v1upgrade")
-  //    .action((_, c) => c.copy(command = Option(V1Upgrade)))
-  //    .text("Automatically upgrade the WDL to version 1.0 and output the result." + System.lineSeparator)
+  cmd("upgrade")
+    .action((_, c) => c.copy(command = Option(Upgrade)))
+    .text("(Experimental) Automatically upgrade the WDL to the latest stable version and output the result." + System.lineSeparator)
```

Comment:
My impression of how this should work (when import support is in place) is one file at a time, rather than converting all of the import targets also.

My reasoning is that if a user has some WDLs that are imported multiple times, they don't want to end up with many duplicates after upgrading WDLs that import them.